src/jace/translator/__init__.py

```diff
@@ -0,0 +1,38 @@
+# JaCe - JAX Just-In-Time compilation using DaCe (Data Centric Parallel Programming)
+#
+# Copyright (c) 2024, ETH Zurich
+# All rights reserved.
+#
+# SPDX-License-Identifier: BSD-3-Clause
+
+"""Subpackage containing all the code related to the Jaxpr to SDFG translation.
+
+The concrete primitive translators that ships with JaCe are inside the `primitive_translators`
+subpackage.
```

Comment:
According to https://stackoverflow.com/a/7948672 package, or subpackage is the correct term.

Reply:
Yes, you're right. I didn't realize the docstring was inide an `__init__.py`. 

By the way, you could have directly checked the proper definition of `package` in the official documentation (https://docs.python.org/3/glossary.html#term-package)  instead of stackoverflow.